pkgs/top-level/perl-packages.nix

```
@@ -20559,6 +20572,20 @@ let
     };
   };
 
+  SubStrictDecl = buildPerlModule {
+    pname = "Sub-StrictDecl";
+    version = "0.005";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/Z/ZE/ZEFRAM/Sub-StrictDecl-0.005.tar.gz";
+      sha256 = "sha256-oSfa52RcGpVwzZopcMbcST1SL/BzGKNKOeQJCY9pESU=";
+    };
+    buildInputs = [ LexicalSealRequireHints ];
+    meta = {
```

I think we need this `perlPreHook` to fix build on Darwin (and perhaps elsewhere:)

```suggestion
    perlPreHook = lib.optionalString stdenv.isDarwin "export LD=$CC";
    meta = {
```